packages/discord.js-utilities/package.json

```
@@ -0,0 +1,59 @@
+{
+	"name": "@sapphire/discord.js-utilities",
+	"version": "1.0.0",
```

```suggestion
	"version": "0.0.1",
```

First major release will be set by Lerna